compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -620,14 +620,26 @@ void addCPULinalgExtTileAndVectorizePipeline(
     OpPassManager &funcPassManager, TilingConfig &tilingConfig,
     LLVMCPUPipelineOptions &pipelineOpt) {
   addTileAndDistributePasses(funcPassManager);
-  funcPassManager.addPass(
-      createLLVMCPUTilePass(tilingConfig.getVectorCommonParallelLevel()));
-  // TODO: Remove the pass once we have PartialReductionOpInterface implemented
-  // for AttentionOp.
+
+  {
+    LLVMCPUTileRootAndFuseProducerConsumerPassOptions options;
+    options.tilingLevel = tilingConfig.getVectorCommonParallelLevel();
+    options.onlyFuseProducerInputOperands = false;
+    funcPassManager.addPass(
```

Comment:
```suggestion
    funcPassManager.addPass(createLLVMCPUTileRootAndFuseProducerConsumer(
      static_cast<int64_t>(tilingConfig.getVectorCommonParallelLevel())))
```

Reply:
I prefer the options being listed one by one, it's more readable